### ISSUE TITLE
Unable to load pancake 1.3 natives when using PHP without session-support

### ISSUE THREAD

**Issue by AgentOak** [pp3345/Pancake#4]
``` bash
$ sudo ./pancake.sh
Found x86_64 processor
Found PHP 5.4
PHP Warning:  PHP Startup: Unable to load dynamic library './natives/core/x86_64_54.so' - ./natives/core/x86_64_54.so: undefined symbol: ps_globals in Unknown on line 0
Pancake natives not loaded. Please run pancake.sh
```

PHP was compiled with

> ./configure --disable-cgi --disable-all --enable-sockets --enable-pcntl --enable-posix --enable-tokenizer --enable-ctype

**Comment by pp3345** [comment]
Not quite a fix, however, it would currently be necessary to compile special binaries for PHP installations without the session extension.

**Comment by pp3345** [comment]
Fixed in 9a5d32372a33d30b8a769ae3a85e38ab87945b4d.
